_episodes/05-ignore.md

```diff
@@ -34,40 +31,29 @@ On branch master
 Untracked files:
   (use "git add <file>..." to include in what will be committed)
 
-	a.dat
-	b.dat
-	c.dat
-	results/
+	.gitignore
+	cats-human-situations.md
+
 nothing added to commit but untracked files present (use "git add" to track)
 ~~~
 {: .output}
 
-Putting these files under version control would be a waste of disk space.
-What's worse,
-having them all listed could distract us from changes that actually matter,
-so let's tell Git to ignore them.
+The file we want to ignore is the `cats-human-situations.md` file. This file has
+been generated by a CSV to Markdown tool, which we found helpful to review
+locally. But it is a derivative of the authoritative CSV file, and others may
+not find it useful. So let's tell Git to ignore it.
 
-We do this by creating a file in the root directory of our project called `.gitignore`:
+We do this by opening the `.gitignore` file in our text editor and adding the name of
+the file below the comments.
 
 ~~~
-$ nano .gitignore
-$ cat .gitignore
-~~~
-{: .bash}
-
-~~~
-*.dat
-results/
+# ...
```

Comment:
What does this bit mean?  Do we even need this code block?

Reply:
Oh, that was to just show putting the line for the file to ignore below the comments. I could just remove that.

Reply:
Oh, that's the contents of `.gitignore`.  I guess that could be more explicit.